desktop/release-notes.md

```diff
@@ -24,35 +24,79 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/general.md/#releases)
 
+## 4.18.0
+
+{% include release-date.html date="2023-03-30" %}
+
+> Download Docker Desktop
+>
+> {% include desktop-install.html %}
+
+### New
+
+- Initial beta release of `docker init` per [docker/roadmap/issues/453](https://github.com/docker/roadmap/issues/453).
+
+### Upgrades
+
+- [Buildx v0.10.4](https://github.com/docker/buildx/releases/tag/v0.10.4)
+- [Compose 2.17.2](https://github.com/docker/compose/releases/tag/v2.17.2)
+- Containerd [v1.6.18](https://github.com/containerd/containerd/releases/tag/v1.6.18), which includes fixes for [CVE-2023-25153](https://github.com/advisories/GHSA-259w-8hf6-59c2) and [CVE-2023-25173](https://github.com/advisories/GHSA-hmfx-3pcx-653p).
+- [criTools v1.26.1](https://github.com/kubernetes-sigs/cri-tools/releases/tag/v1.26.1)
+- [cri-dockerd v0.3.1](https://github.com/Mirantis/cri-dockerd/releases/tag/v0.3.1)
+- [coredns v1.9.3](https://github.com/coredns/coredns/releases/tag/v1.9.3)
+- [cni v1.2.0](https://github.com/containernetworking/plugins/releases/tag/v1.2.0)
+- [Kubernetes v1.26.3](https://github.com/kubernetes/kubernetes/releases/tag/v1.26.3)
+
+### Bug fixes and enhancements
+
+- [Docker Scout CLI](https://docs.docker.com/scout/#docker-scout-cli) can compare two images and display packages and vulnerabilities differences. This command is in early preview and might change in the future.
+- [Docker Scout CLI](https://docs.docker.com/scout/#docker-scout-cli) now displays base image update and remediation recommendations using `docker scout recommendations` and an short overview of an image using `docker scout quickview` commands.
+- You can now search for extensions in **Global Search** and in the Marketplace.
+- Fix bug where `docker buildx` container builders would lose access to the network after 24hrs.
+- Reduced how often users are prompted for feedback on Docker.
+- Removed minimum VM swap size.
+- Added a new Learning Center tab to help users get started with Docker.
+- Add support for subdomain match, CIDR match, `.` and `_.` in HTTP proxy exclude lists.
+- Fix bug in the transparent TLS proxy when the Server Name Indication field is not set.
+- Fixed a grammatical error in Docker Desktop engine status message.
+
+### For Windows
+
+- Fixed a bug where `docker run --gpus=all` hangs. Fixes [docker/for-win#13333](https://github.com/docker/for-win/issues/13333).
+- Add support for enhanced container isolation in Windows hosts with WSL.
```

Comment:
I think this needs to be removed? I think it's being shipped in 4.19 instead. @MihaelaStoica @christophermclellan can you confirm?

Reply:
Yes, it needs removed. Thanks for spotting it!